README.md

```
@@ -274,6 +274,7 @@
   #### OpenSource Other Bots
   - __[English Wikipedia Link Converter](https://github.com/jnton/english-wikipedia-link-converter-telegram-bot)__  By [JnTon](https://github.com/jnton) : _Telegram bot that converts any non-English Wikipedia link into its English equivalent_
   - __[freqtrade](https://github.com/freqtrade/freqtrade)__  By [freqtrade](https://github.com/freqtrade) : _Free, open source crypto trading bot_
+  - __[jellyfin-telegram-channel-sync](https://github.com/GeiserX/jellyfin-telegram-channel-sync)__  By [GeiserX](https://github.com/GeiserX) : _Syncs Jellyfin user access with Telegram channel membership, automatically disabling accounts when members leave._
```

markdownlint-cli2 (0.21.0)

[warning] Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)